vscode/src/extension.ts

```diff
@@ -184,6 +184,36 @@ async function activateLanguageService(extensionUri: vscode.Uri) {
     ...registerQSharpNotebookCellUpdateHandlers(languageService),
   );
 
+  // Watch manifest changes and update each document in the same project as the manifest.
+  subscriptions.push(
+    vscode.workspace.onDidSaveTextDocument((manifest) => {
```

Comment:
Do you want to listen to `onDidDeleteFiles` as well? Similar to this, I think we'd want to trigger an update on all the child .qs files so they can get reparented to single-file-projects.

Reply:
Good observation. I just added it!